content/vgg-classifier-tutorial.md

```diff
@@ -0,0 +1,320 @@
+Title: Keras, Make your own image classifier
+Date: 2016-03-30
+Category: Tutorial
+Author: Gregory Senay
+
+
+This tutorial explains how to create image classifier, from an ImageNet pre-trained model, your own model on your own dataset.
+
+Prepare your dataset
+-----
+Here, I will explain how to simply prepare your image dataset following an existing simple one: Caltech101.
+Caltech101 is a dataset of 101 categories (crocodile, camera, plane, soccer ball...) with 40 to 800 images per category, the size of image is bout 200x300.
+To prepare your data, you have to follow this directory architecture:
+```
+  * Main Directory # Name of you dataset
+    * Class1Name
+      * Image1Class1
+      * Image2Class1
+      * Image3Class1
+      * ...
+    * Class2Name
+      * Image1Class1
+      * Image2Class1
+      * ...
+    * Class3Name
+      * Image1Class1
+      * Image2Class1
+      * ...
+    * Class4Name
+    * ...
+
+```
+
+In the case of Caltech101, (available here: http://www.vision.caltech.edu/Image_Datasets/Caltech101/101_ObjectCategories.tar.gz), once you extract the archive with ``tar xvf 101_ObjectCategories.tar.gz)`` you have this directory architecture:
+
+``
+ * 101_ObjectCategories
+ 	* Faces
+		 * image_0001.jpg
+ 		 * image_0002.jpg
+		 * image_0003.jpg
+		 * ...
+ 	* Leopards
+		* image_0001.jpg
+ 		* image_0002.jpg
+		* ...
+ 	* Motorbikes
+		* image_0001.jpg
+ 		* image_0002.jpg
+		* ...
+ 	* accordion
+	* ...
+``
+
+Advice: if you create your own classes, try to have a minimum of 40 images with a lot of variabilities (i.e. if you have a cat category try to have different cat positions and different cat type, color...).
+
+Unfortunately, the model you are using below requires a same dimension for all of the images, so we need to load and harmonize the image dataset.
```

Comment:
"the model we will be using here requires all input images to have a specific size, 224x224".

We should actually remove the FC layers from the original network to make it possible to train on any image size (as long as all new images are the same size). It is fine to do that because in the example below you are deleting the last logreg layer anyway.